src/emmy/collection.cljc

```diff
@@ -235,33 +220,23 @@
   (cs/union a b))
 
 (doseq [klass [PersistentHashSet PersistentTreeSet]]
+  (defmethod g/zero? [klass] [s] (empty? s))
```

Comment:
I'm counting on our tests to catch my tired eyes as I read these implementations... this is heroic work!

Reply:
"Refactor in the presence of running tests" has never felt so relevant!!